cli/mobycli/exec.go

```diff
@@ -111,9 +111,18 @@ func Exec(_ *cobra.Command) {
 	}
 	commandArgs := os.Args[1:]
 	command := metrics.GetCommand(commandArgs)
-	if command == "login" && !metrics.HasQuietFlag(commandArgs) {
-		displayPATSuggestMsg(commandArgs)
+	if !metrics.HasQuietFlag(commandArgs) {
+		switch command {
+		case "build": // only on regular build, not on buildx build
```

Comment:
`build` command redirects to `buildx build` since Docker 23 and I don't see in the `metrics.HasQuietFlag` anything checking the BuildKit progress quiet mode.